lib/schedule/minischedule/trip.ex

```diff
@@ -47,7 +51,9 @@ defmodule Schedule.Minischedule.Trip do
       via_variant: trip.route_pattern_id && RoutePattern.via_variant(trip.route_pattern_id),
       run_id: trip.run_id,
       start_time: trip.start_time,
-      end_time: trip.end_time
+      end_time: trip.end_time,
+      start_place: Map.get(timepoint_names_by_id, trip.start_place),
+      end_place: Map.get(timepoint_names_by_id, trip.end_place)
```

Comment:
I guess the designs don't say to fall back to the id if we can't find the name, but it seems like it might be good to give inspectors whatever information we have. Ids are usually almost readable, and inspectors are familiar with the id format from the ladders. But I see how it could be confusing.

Reply:
@ingridpierre what do you think?

Reply:
IDs would be "cabot" vs "Cabot Garage"? I think that's acceptable. 

Blanks do little for us here, and I'd like something rather than nothing so that I can check against Transit Master and the schedule to make sure I'm seeing what I should be (even if the name doesn't look the prettiest).